print_nanny_webapp/alerts/models.py

```diff
@@ -310,6 +358,40 @@ def trigger_email_alert(self, data):
 
         return message
 
+    def trigger_discord_alert(self, data):
```

Comment:
Check out my above comment about Django channels! Basically, this method's job should be to serialize/push a message to a channel layer.